JavaAssignments/src/mayur/SwapNumbers.java

```diff
@@ -0,0 +1,21 @@
+package mayur;
+
+public class SwapNumbers {
+
+	public static void main(String args[]) 
+	{
+		float Num1 = (float) Double.parseDouble(args[0]);
```

Comment:
If you want to accept numbers in float then you should Float.parseFloat(args[0])